doc/man7/provider-cipher.pod

```diff
@@ -197,18 +197,18 @@ parameters are relevant to, or are understood by all ciphers:
 
 =over 4
 
-=item B<OSSL_CIPHER_PARAM_PADDING> (int)
+=item B<OSSL_CIPHER_PARAM_PADDING> (size_t)
 
 Sets the padding mode for the associated cipher ctx.
 Setting a value of 1 will turn padding on.
-Setting a vlue of 0 will turn padding off.
+Setting a value of 0 will turn padding off.
```

Comment:
How did all these typos make it past the review stage?????

Reply:
You must of reviewed it :)